### PR TITLE
.NET Core support

### LINKED ISSUE #50: .NET Core support
adding support for .NET Core

### PR DESCRIPTION
**Pull request by lahma**
* target .NET 4.5.2, netstandard1.3 and netstandard2.0

### CODE REVIEW

File: RserveCLI2.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.26114.2
```
Comment (lahma) [inline RserveCLI2.sln]: Visual Studio 2017.3 / NET Core 2.0 SDK required due to netstandard2.0 support

**Comment by konne** [comment]
@lahma thx, at the moment the automatic builds and test are not working so I will test this manuall before I merge your great contribution, I try to get it done ASAP

**Comment by lahma** [comment]
@konne sure, no problem. I believe Appveyor hasn't yet updated their default image, so you need to use preview one that has the .NET Core 2 baked in. [Here's an example](https://github.com/quartznet/quartznet/blob/master/appveyor.yml) , I'd recommend having same style config in version control, with required modifications of course.

**Comment by konne** [comment]
@lahma in the first step I will remove appv. and will come back maybe later with travisci, because I like to have one product for all my projects. Second I like the idea to test in travis with autmated installed R and not with an R that is included in the GIT repo like now

**Comment by lahma** [comment]
@konne so a year has passed, how does this PR's future look like? :smile:

**Comment by konne** [comment]
@lahma thanks for the reminder, I will take care

**Comment by konne** [comment]
@lahma I published a RserveCLI2.0.1.0-alpha0102 out of the development branch, can you please try to use this and report issues. I will add in the next days also travis ci docker based tests like: https://github.com/senseyeio/roger/tree/master/test

**Comment by lahma** [comment]
Great! Thank you. I'll give it a spin.

**Comment by konne** [comment]
@lahma could you already test the alpha nuget?

**Comment by lahma** [comment]
We are running a fork rebased over this (more aggressive optimizations) and seems to work fine with our data sets.

**Comment by konne** [comment]
is there any reason not to include your optimizations into the main repo. I like to get this repo slowly updated to a state of the art repo

**Comment by lahma** [comment]
Sure, I can prepare a pull request containing the relevant bits when I have the time.

**Comment by konne** [comment]
@lahma thanks for your last SourceLink PR. But I still like to ask again for your PR with the relevant bits

**Pull request merged**